### ISSUE TITLE
Default labels for groups and nodes

### ISSUE THREAD

**Issue by kdungs** [blushft/go-diagrams#12]
Hello there,

first of all thanks a lot for this library. :)

tl;dr: i've found issues with the default labels for groups and prepared a corresponding PR.

When trying it out, I found some issues with default labels, that can cause problems for newcomers especially if they don't know anything about graphviz.

The default label for a group is the empty string. When rendered, it produces `label=;` which to graphviz is a syntax error. So, in order to produce a valid graph, at least the `diagram.Label` option is needed, as far as I see it. If I'm not mistaken, this could be fixed by adding a call to `trimAttrs` here: https://github.com/blushft/go-diagrams/blob/master/diagram/options.go#L39

```Go
d, err := diagram.New(
	diagram.Label("TEST"),
)
```

The default label for a node is "node". This is a reserved keyword in graphviz and when left unescaped causes problems. Since graphviz has logic for automatically labelling nodes, I would suggest using empty string here, as well and letting graphviz set the label. The alternative would be something like `"\"node\""` which doesn't feel right to me. But feel free to suggest that on the PR if you see it differently.
